From 68afb0c8f61b9861e903be35e385accd9bdbe93c Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Wed, 25 May 2011 00:14:04 +0800
Subject: [PATCH 983/998] ENGR00144003 mx3: Fix build failure issue due to mc34708 driver

This is just one workaround to enable MC34708 for MX3
platform to bypass build failure issue.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/configs/mx3_defconfig         |   26 +++++++++++++++++++++++---
 arch/arm/configs/mx3_updater_defconfig |   10 ++++++++--
 2 files changed, 31 insertions(+), 5 deletions(-)

diff --git a/arch/arm/configs/mx3_defconfig b/arch/arm/configs/mx3_defconfig
index ca26bc4..bca30dc 100644
--- a/arch/arm/configs/mx3_defconfig
+++ b/arch/arm/configs/mx3_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
-# Tue Sep  7 14:59:08 2010
+# Wed May 25 00:10:15 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -192,6 +192,7 @@ CONFIG_MMU=y
 # CONFIG_ARCH_FOOTBRIDGE is not set
 CONFIG_ARCH_MXC=y
 # CONFIG_ARCH_STMP3XXX is not set
+# CONFIG_ARCH_MXS is not set
 # CONFIG_ARCH_NETX is not set
 # CONFIG_ARCH_H720X is not set
 # CONFIG_ARCH_IOP13XX is not set
@@ -402,13 +403,13 @@ CONFIG_HAVE_AOUT=y
 #
 CONFIG_PM=y
 CONFIG_PM_DEBUG=y
-CONFIG_SUSPEND_DEVICE_TIME_DEBUG=y
 # CONFIG_PM_ADVANCED_DEBUG is not set
 # CONFIG_PM_VERBOSE is not set
 CONFIG_CAN_PM_TRACE=y
 CONFIG_PM_SLEEP=y
 CONFIG_SUSPEND_NVS=y
 CONFIG_SUSPEND=y
+CONFIG_SUSPEND_DEVICE_TIME_DEBUG=y
 CONFIG_SUSPEND_FREEZER=y
 # CONFIG_APM_EMULATION is not set
 # CONFIG_PM_RUNTIME is not set
@@ -578,6 +579,7 @@ CONFIG_MTD_MXC=y
 #
 # CONFIG_MTD_DATAFLASH is not set
 # CONFIG_MTD_MXC_DATAFLASH is not set
+# CONFIG_MTD_MXC_M25P80 is not set
 # CONFIG_MTD_M25P80 is not set
 # CONFIG_MTD_SST25L is not set
 # CONFIG_MTD_SLRAM is not set
@@ -715,6 +717,7 @@ CONFIG_DNET=y
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_FEC=y
+# CONFIG_FEC_1588 is not set
 # CONFIG_FEC2 is not set
 # CONFIG_NETDEV_1000 is not set
 # CONFIG_NETDEV_10000 is not set
@@ -940,11 +943,14 @@ CONFIG_MFD_WM8350_CONFIG_MODE_0=y
 CONFIG_MFD_WM8352_CONFIG_MODE_0=y
 CONFIG_MFD_WM8350_I2C=y
 # CONFIG_MFD_WM8994 is not set
+# CONFIG_MFD_LTC3589_I2C is not set
 # CONFIG_MFD_PCF50633 is not set
 # CONFIG_MFD_MC13783 is not set
+# CONFIG_PMIC_DA9052 is not set
 # CONFIG_ABX500_CORE is not set
 # CONFIG_EZX_PCAP is not set
 # CONFIG_AB8500_CORE is not set
+# CONFIG_MFD_MAX17135 is not set
 CONFIG_REGULATOR=y
 # CONFIG_REGULATOR_DEBUG is not set
 # CONFIG_REGULATOR_DUMMY is not set
@@ -960,8 +966,8 @@ CONFIG_REGULATOR_WM8350=y
 # CONFIG_REGULATOR_TPS65023 is not set
 # CONFIG_REGULATOR_TPS6507X is not set
 CONFIG_REGULATOR_MC13892=y
+CONFIG_REGULATOR_MC34708=y
 CONFIG_REGULATOR_MC9S08DZ60=y
-# CONFIG_REGULATOR_MAX17135 is not set
 CONFIG_MEDIA_SUPPORT=y
 
 #
@@ -1008,8 +1014,11 @@ CONFIG_VIDEO_MXC_CAMERA=y
 # CONFIG_MXC_CAMERA_MICRON111 is not set
 # CONFIG_MXC_CAMERA_OV2640 is not set
 # CONFIG_MXC_CAMERA_OV3640 is not set
+# CONFIG_MXC_CAMERA_OV5640 is not set
+# CONFIG_MXC_CAMERA_OV5642 is not set
 # CONFIG_MXC_TVIN_ADV7180 is not set
 CONFIG_VIDEO_MXC_OUTPUT=y
+# CONFIG_VIDEO_MXC_OPL is not set
 # CONFIG_VIDEO_SAA5246A is not set
 # CONFIG_VIDEO_SAA5249 is not set
 CONFIG_SOC_CAMERA=y
@@ -1143,6 +1152,7 @@ CONFIG_DMA_ENGINE=y
 CONFIG_MXC_PMIC=y
 # CONFIG_MXC_PMIC_MC13783 is not set
 CONFIG_MXC_PMIC_MC13892=y
+CONFIG_MXC_PMIC_MC34708=y
 CONFIG_MXC_PMIC_I2C=y
 # CONFIG_MXC_PMIC_SPI is not set
 # CONFIG_MXC_PMIC_MC34704 is not set
@@ -1158,12 +1168,17 @@ CONFIG_MXC_MC13892_LIGHT=y
 # CONFIG_MXC_MC13892_BATTERY is not set
 # CONFIG_MXC_MC13892_CONNECTIVITY is not set
 CONFIG_MXC_MC13892_POWER=y
+# CONFIG_MXC_MC34708_ADC is not set
+# CONFIG_MXC_MC34708_RTC is not set
+# CONFIG_MXC_MC34708_BATTERY is not set
 CONFIG_MXC_PMIC_MC9S08DZ60=y
 # CONFIG_MXC_MC9SDZ60_RTC is not set
 
 #
 # MXC Security Drivers
 #
+# CONFIG_MXC_SECURITY_SCC is not set
+# CONFIG_MXC_SECURITY_RNG is not set
 
 #
 # MXC MPEG4 Encoder Kernel module support
@@ -1208,6 +1223,11 @@ CONFIG_MXC_VPU=y
 # CONFIG_IMX_ADC is not set
 
 #
+# MXC GPU support
+#
+# CONFIG_MXC_AMD_GPU is not set
+
+#
 # File systems
 #
 # CONFIG_EXT2_FS is not set
diff --git a/arch/arm/configs/mx3_updater_defconfig b/arch/arm/configs/mx3_updater_defconfig
index f589634..3f779b4 100644
--- a/arch/arm/configs/mx3_updater_defconfig
+++ b/arch/arm/configs/mx3_updater_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
-# Sun Sep 26 12:30:14 2010
+# Wed May 25 00:22:11 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -399,6 +399,7 @@ CONFIG_CAN_PM_TRACE=y
 CONFIG_PM_SLEEP=y
 CONFIG_SUSPEND_NVS=y
 CONFIG_SUSPEND=y
+# CONFIG_SUSPEND_DEVICE_TIME_DEBUG is not set
 CONFIG_SUSPEND_FREEZER=y
 # CONFIG_APM_EMULATION is not set
 # CONFIG_PM_RUNTIME is not set
@@ -484,6 +485,7 @@ CONFIG_MTD_MXC=y
 #
 # CONFIG_MTD_DATAFLASH is not set
 # CONFIG_MTD_MXC_DATAFLASH is not set
+# CONFIG_MTD_MXC_M25P80 is not set
 # CONFIG_MTD_M25P80 is not set
 # CONFIG_MTD_SST25L is not set
 # CONFIG_MTD_SLRAM is not set
@@ -766,8 +768,8 @@ CONFIG_REGULATOR=y
 # CONFIG_REGULATOR_TPS65023 is not set
 # CONFIG_REGULATOR_TPS6507X is not set
 CONFIG_REGULATOR_MC13892=y
+CONFIG_REGULATOR_MC34708=y
 CONFIG_REGULATOR_MC9S08DZ60=y
-# CONFIG_REGULATOR_MAX17135 is not set
 # CONFIG_MEDIA_SUPPORT is not set
 
 #
@@ -936,6 +938,7 @@ CONFIG_DMADEVICES=y
 CONFIG_MXC_PMIC=y
 # CONFIG_MXC_PMIC_MC13783 is not set
 CONFIG_MXC_PMIC_MC13892=y
+CONFIG_MXC_PMIC_MC34708=y
 CONFIG_MXC_PMIC_I2C=y
 # CONFIG_MXC_PMIC_SPI is not set
 # CONFIG_MXC_PMIC_MC34704 is not set
@@ -951,6 +954,9 @@ CONFIG_MXC_MC13892_LIGHT=y
 # CONFIG_MXC_MC13892_BATTERY is not set
 # CONFIG_MXC_MC13892_CONNECTIVITY is not set
 CONFIG_MXC_MC13892_POWER=y
+# CONFIG_MXC_MC34708_ADC is not set
+# CONFIG_MXC_MC34708_RTC is not set
+# CONFIG_MXC_MC34708_BATTERY is not set
 CONFIG_MXC_PMIC_MC9S08DZ60=y
 # CONFIG_MXC_MC9SDZ60_RTC is not set
 
-- 
1.7.4.1

